src/components/checkout/TrainTicketSelection.tsx

```diff
@@ -94,14 +99,64 @@ export const TrainTicketSelection = () => {
           />
         </div>
         <div className="TrainSeatBox">
+          
           <FormControlLabel
             value="choose-seat"
             control={<Radio />}
             label="Choose certain reservation"
           />
+        
           <CreditCardIcon />
         </div>
-      </RadioGroup>
+        </RadioGroup>
+
+        <RadioGroup
```

Comment:
1. they should be shown only when second radio button active
![image](https://user-images.githubusercontent.com/81109647/210855980-29933217-e186-4760-ae78-26fd681b30e3.png)

2. that should be checkboxes instead of radio buttons and they should be shifted to the right a bit